### PR TITLE
fix(bundling): switch to using umd bundles

### LINKED ISSUE #456: Zone.js browser bundle pollutes global namespace
The `browser.ts` file (https://github.com/angular/zone.js/blob/master/lib/browser/browser.ts) is not being bundled inside any closure, resulting in all of its top-level functions and variables becoming globals.

This was discovered because this behaviour was breaking Plunker's preview by overriding `window.name` to `alert`. Down the rabbit hole we go and find ourselves here.

### PR DESCRIPTION
**Pull request by juliemr**
This means that we will stop polluting the global namespace when
zone.js is included.

Fixes #456

### CODE REVIEW

**Comment by juliemr** [comment]
note: I've included the updates from running `npm run prepublish` here because it makes the review more meaningful.

Let me know if you'd like me to revert the changes to stuff in `dist/`

File: dist/async-test.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
+(function (global, factory) {
```
Comment (mhevery) [inline dist/async-test.js]: We should add copyright here:

```
/**
 * @license
 * Copyright Google Inc. All Rights Reserved.
 *
 * Use of this source code is governed by an MIT-style license that can be
 * found in the LICENSE file at https://angular.io/license
 */
```

Comment (mhevery) [inline dist/async-test.js]: Also while it is nice to see final product, do revert `dist` folder. before checking in.

Comment (mhevery) [inline dist/async-test.js]: We should add copyright here:

```
/**
 * @license
 * Copyright Google Inc. All Rights Reserved.
 *
 * Use of this source code is governed by an MIT-style license that can be
 * found in the LICENSE file at https://angular.io/license
 */
```

Comment (mhevery) [inline dist/async-test.js]: Also while it is nice to see final product, do revert dist folder. before checking in.

File: dist/async-test.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+    typeof exports === 'object' && typeof module !== 'undefined' ? factory() :
+    typeof define === 'function' && define.amd ? define(factory) :
+    (factory());
+}(this, (function () { 'use strict';
+
 (function () {
```
Comment (mhevery) [inline dist/async-test.js]: All of the specs have their own closure which auto-detects global. We should remove these?

Comment (juliemr) [inline dist/async-test.js]: Done

File: dist/async-test.js (hunk 1 above)
Comment (juliemr) [inline dist/async-test.js]: Added copyright.

ACK for reverting dist folder, will do before checking in.

**Comment by juliemr** [comment]
Responded to comments and updated the dist files - I'll squash and revert the dist files before merging if this looks good to you.

**Pull request merged**